src/Stratis.Bitcoin.Features.ColdStaking/ColdStakingFeature.cs

```diff
@@ -232,6 +238,8 @@ public static IFullNodeBuilder UseColdStakingWallet(this IFullNodeBuilder fullNo
 
             LoggingConfiguration.RegisterFeatureNamespace<ColdStakingFeature>("wallet");
 
+            fullNodeBuilder.UseWallet();
```

Comment:
Another option would be to create an override to the wallet feature that allows to provide a manager `fullNodeBuilder.UseWallet<ColdStakingManager>(); `

Reply:
Yes. Many ways to do this.